klogr/klogr.go

```diff
@@ -76,7 +121,12 @@ func flatten(kvList ...interface{}) string {
 	}
 	sort.Strings(keys)
 	buf := bytes.Buffer{}
+	seenKeys := make(map[string]interface{}, len(keys))
 	for i, k := range keys {
+		if _, ok := seenKeys[k]; ok {
+			continue
+		}
+		seenKeys[k] = k
```

Comment:
This handling shouldn't be required now as we already deduplicate in `trimDuplicates` - should I remove this from here? It seems a bit redundant 😄

Reply:
yep, go ahead